client/package.json

```diff
@@ -30,13 +30,13 @@
     "vuex-class": "^0.3.2"
   },
   "devDependencies": {
-    "@types/jest": "^24.9.0",
+    "@types/jest": "^25.1.4",
     "@types/sinon": "^7.5.1",
-    "@vue/cli-plugin-babel": "^4.0.0",
+    "@vue/cli-plugin-babel": "^4.2.3",
```

Comment:
It's actually already using v4.4.1 (according to package lock file).

Reply:
Also "@vue/cli-plugin-unit-jest" below.

Reply:
Ah! Confusing. Why is this the case?

Reply:
The caret at the front of the semantic version indicates that it can auto-update to anything up to "4.x.y".

Note that we check in the package lock file to specify exactly which versions are used for the build (otherwise it would auto-update).

This gets updated when you do a `npm i` I think.

You can check the package lock file to see the exact versions that are used.

Reply:
^^ We don't want the build server to auto-update. We want it to use exactly the same package versions that we use locally.

Reply:
How should we approach this. Update all the versions to the ones actually used according to the package-lock?

Reply:
Only if you are as AR as me. And it will gradually get out of sync again anyway (which should have no impact except to legibility).
So,  maybe update it this time then fuggedaboutit.

Reply:
😆 😆  Alrighty!